### DISCUSSION TITLE
Could not poll the database: SQLSTATE[42S02]

### DISCUSSION THREAD

**Discussion by Dvalin21** [firefly-iii/firefly-iii#7806] [Firefly III] [answered]
### Support guidelines

- [X] I've read the [support guidelines](https://github.com/firefly-iii/firefly-iii/blob/main/.github/support.md)

### I've found a bug and checked that ...

- [X] ... [the documentation](https://docs.firefly-iii.org/) does not mention anything about my problem
- [X] ... there are no open or closed issues that are related to my problem
- [X] ... it's [definitely a Firefly III issue, not me](https://github.com/firefly-iii/firefly-iii/blob/main/.github/its_you_not_me.md)

### Description

Installed FireFly III and keep getting this error about 1146 Table.

### Debug information

Could not poll the database: SQLSTATE[42S02]: Base table or view not found: 1146 Table 'firefly.configuration' doesn't exist (Connection: mysql, SQL: select `id`, `name`, `data` from `configuration` where `name` = is_demo_site and `configuration`.`deleted_at` is null limit 1)

This error occured in file /var/www/html/app/Support/FireflyConfig.php on line 80 with code 0.

% Total % Received % Xferd Average Speed Time Time Time Current Dload Upload Total Spent Left Speed 0 0 0 0 0 0 0 0 --:--:-- --:--:-- --:--:-- 0 0 0 0 0 0 0 0 0 --:--:-- --:--:-- --:--:-- 0 curl: (22) The requested URL returned error: 500 Internal Server Error

### Expected behaviour

I expected FireFlyIII to start for the first time, taking me through the setup process.

### Steps to reproduce

Installing FireFly III

**Comment by Dvalin21** [comment, marked as answer]
So never mind, but ill leave this here in the event someone else has the same issue. If your installing on a Synology Nas. Make sure to take docker ip for the db and FireFly container and add them to your firewall for allow. Otherwise your not getting anything done.

**Reply by anonymousheadless** [reply]
Could you elaborate on this, as I'm getting this issue with docker compose on Windows, its some setting because it seems to lock out the db with one of its init settings, as I could login into the db before starting core but after that my connection is refused.

**Reply by Dvalin21** [reply]
The issue that I was getting was a firewall issue. To be more specific, the docker ip and/or port is being blocked with is why I was getting this issue. It could be the same issue for you in windows.

**Comment by JC5** [comment]
A question AND a solution in one go, nice!
